modules/vba_documents/spec/request/upload_complete_request_spec.rb

```diff
@@ -40,12 +40,15 @@
 
     context 'verified message' do
       it 'should confirm the subscription' do
-        with_settings(Settings.vba_documents.sns, 'topic_arn' => 'arn:aws:sns:us-west-2:123456789012:MyTopic') do
+        with_settings(Settings.vba_documents.sns,
+                      'topic_arn' => 'arn:aws:sns:us-west-2:123456789012:MyTopic',
+                      'region' => 'us-gov-west-1') do
           client = double(Aws::SNS::Client)
           expect(client).to receive(:confirm_subscription).with(
-            topic_arn: 'arn:aws:sns:us-west-2:123456789012:MyTopic',
+            authenticate_on_unsubscribe: 'authenticateOnUnsubscribe',
             token: token,
-            authenticate_on_unsubscribe: 'authenticateOnUnsubscribe'
+            topic_arn: 'arn:aws:sns:us-west-2:123456789012:MyTopic',
```

Comment:
Does this line need the region updated?

Reply:
I could update the region names in the mock SNS notifications I'm using, but it should just be passing the topic arn from that. The mock notifications are just taken from the AWS docs.